style.css

```diff
@@ -0,0 +1,1427 @@
+  }
+
+  .news-img-8 {
+    grid-area: 3 / 2 / 4 / 3;
+  }
+
+  .section-heading {
+    font-size: 85%;
+  }
+
+  .info-card-768 {
+    display: none;
+  }
+
+  .info-cards {
+    width: auto;
+  }
+
+  .great-mid {
+    width: 90vw;
+    height: auto;
+    margin: auto;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr 1fr 1fr;
+    grid-template-areas:
+      "hard apps"
+      "gear tech"
+      "creative podium"
+      "insights launch"
+      "distract . ";
+    margin-bottom: 8vh;
+  }
+
+  .ld-head {
+    display: flex;
+    width: 90vw;
+    height: auto;
+    margin: auto;
+    justify-content: space-between;
+    padding-bottom: 15px;
+    font-size: 60%;
+  }
+
+  .latest-deals {
+    width: 95vw;
+    height: auto;
+    margin: auto;
+    display: grid;
+    grid-template-columns: 30vw 30vw 30vw;
+    grid-template-rows: 1fr 1fr 1fr;
+    margin-bottom: 8vh;
+  }
+
+  .ld-img-1 {
+    grid-area: 1 / 1 / 2 / 2;
+  }
+
+  .ld-img-2 {
+    grid-area: 1 / 2 / 2 / 3;
+  }
+
+  .ld-img-3 {
+    grid-area: 1 / 3 / 2 / 4;
+  }
+
+  .ld-img-4 {
+    grid-area: 2 / 1 / 3 / 2;
+  }
+
+  .ld-img-5 {
+    grid-area: 2 / 2 / 3 / 3;
+  }
+
+  .ld-img-6 {
+    grid-area: 2 / 3 / 3 / 4;
+  }
+
+  .ld-img-7 {
+    grid-area: 3 / 1 / 4 / 2;
+  }
+
+  .ld-img-8 {
+    grid-area: 3 / 2 / 4 / 3;
+  }
+
+  .ld-img-1,
+  .ld-img-2,
+  .ld-img-3,
+  .ld-img-4,
+  .ld-img-5,
+  .ld-img-6,
+  .ld-img-7,
+  .ld-img-8 {
+    font-size: 80%;
+  }
+
+  footer {
+    width: 99vw;
+  }
+}
+
+@media screen and (max-width: 767px) {
+  .nav-container {
+    display: none;
+  }
+
+  .mobile-top-nav {
+    display: flex;
+    justify-content: space-between;
+    top: 0;
+    height: 12vh;
+    width: 98vw;
+    position: fixed;
+    z-index: 2;
+    background-color: rgb(253, 253, 253);
+  }
+
+  .container-768-mobile {
+    background: black;
+    height: 100%;
+    width: 25%;
+    display: flex;
+    justify-content: center;
+  }
+
+  .logo-for-768-logo-mobile {
+    height: 100%;
+    width: 30%;
+    display: flex;
+    justify-content: center;
+  }
+
+  .fa-s {
+    display: flex;
+    padding-left: 10px;
+    text-decoration: none;
+  }
+
+  .info-cards a {
+    text-decoration: none;
+  }
+
+  .fa-s > div > a {
+    text-decoration: none;
+  }
+
+  .mobile-top-nav i {
+    display: block;
+    height: 48px;
+    opacity: 0.25;
+    width: 30px;
+    padding-top: 4vh;
+  }
+
+  .section-one {
+    font-size: 40%;
+    display: grid;
+    width: 100vw;
+    height: 80vh;
+    grid-template-columns: 96vw;
+    grid-template-rows: 25vh 25vh 25vh;
+    margin-top: 12vh;
+  }
+
+  .first {
+    grid-area: 1 / 1 / 2 / 2;
+    background-image:
+      linear-gradient(0deg, rgba(0, 0, 0, 1) 0%, rgba(255, 255, 255, 0) 50%),
+      url("./images/Opal-796x417.jpg");
+  }
+
+  .second {
+    grid-area: 2 / 1 / 3 / 2;
+  }
+
+  .third {
+    grid-area: 3 / 1 / 4 / 2;
+  }
+
+  .tnw-logo {
+    display: none;
+  }
+
```

Comment:
## Aim to Stay `DRY`
- [ ] The lines of code on line `#1097` & `#1174` appear to be similar and make you seem like you are repeating yourself. Would you mind merging them so as to not appear redundant?